lib/core/elastic_connector_actions.rb

```diff
@@ -88,6 +88,16 @@ def claim_job(connector_package_id)
         job['_id']
       end
 
+      def heartbeat(connector_package_id)
```

Comment:
nit - the new taxonomy suggests we shouldn't use `connector_package` but `connector_client`

Reply:
Good one, I will actually rename `connector_package_*` to `connector_client_*` in a separate PR.